cypress/page_objects/registerPage.js

```diff
@@ -24,28 +24,28 @@ export default class RegisterPage {
         return cy.get(`select[formcontrolname=${name}]`);
     }
 
-    registerNew(firstName, lastName, email, password, passwordConfirmation, telephone, id, city, address, postalcode, terms){
+    registerNew(element){
         cy.wait(3000);
-        firstName
-            ?this.getInputField('text').eq(0).type(firstName)
+        element.firstName
+            ?this.getInputField('text').eq(0).type(element.firstName)
```

Comment:
Ovo moze i uz pomoc If da se uradi u slucaju da "else" deo nikada ne izvrsavamo smisleno. Neka ostane kako je sada, to smo vec prodiskutovali.